todo_app/app.py

```diff
@@ -1,27 +1,11 @@
-from flask import Flask, render_template
+from flask import Flask
 
 from todo_app.flask_config import Config
 
-from todo_app.data.session_items import get_items
-
-from todo_app.data.session_items import add_item
-
-from flask import request, redirect, session
-
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    todolist = get_items()
-    titles = [d['title'] for d in todolist]
-    return render_template ('index.html', list1=titles)
-
-@app.route('/additem', methods=['GET', 'POST'])
-def additem():
-    if request.method =='POST':
-        title = request.form.get('title1')
-        add_item(title)
-        return redirect (('/'))
```

Comment:
Given that you only allow POST requests in the body, we might as well only allow POST requests full stop!
```python
@app.route('/additem', methods=['POST'])
def additem():
      title = request.form.get('title1')
      add_item(title)
      return redirect (('/'))
```